docs/UserGuide.md

```diff
@@ -20,7 +20,7 @@ reliably than traditional GUI based apps.
 ___
 
 ## Table of Contents
-- Table of Contents {:toc}
+* Table of Contents {:toc}
```

Comment:
Actually this line still doesn't generate the table correctly, the {:toc} should be on the next line.